administrator/components/com_admin/sql/updates/mysql/4.1.0-2021-07-16.sql

```diff
@@ -0,0 +1,51 @@
+--
+-- Table structure for table `#__guidedtours`
+--
+
+CREATE TABLE IF NOT EXISTS `#__guidedtours` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `asset_id` int DEFAULT 0,
+  `title` varchar(255) NOT NULL,
+  `alias` varchar(400) CHARACTER SET utf8mb4 COLLATE utf8mb4_bin NOT NULL,
+  `description` text NOT NULL,
+  `ordering` int NOT NULL DEFAULT 0,
+  `created` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
+  `created_by` int NOT NULL DEFAULT 0,
+  `modified` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP(),
+  `modified_by` int NOT NULL DEFAULT 0,
+  `checked_out_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP() ,
+  `checked_out` int NOT NULL DEFAULT 0,
+  `published` tinyint(4) NOT NULL DEFAULT 0,
+  `state` tinyint(4) NOT NULL DEFAULT '1',
```

Comment:
```suggestion
  `published` tinyint NOT NULL DEFAULT 0,
  `state` tinyint NOT NULL DEFAULT '1',
```
We have removed the display widths from all integer data types in MySQL due to the reasons described here https://github.com/joomla/joomla-cms/pull/32606 . They have zero impact on storage space or value range and are deprecated in MySQL 8.